app/views/renewal/percentage.scala.html

```diff
@@ -22,32 +22,30 @@
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @heading("renewal.hvd.percentage.title", "summary.renewal")
+    @heading(section = "summary.renewal")
 }
 
 @main(
     title = Messages("renewal.hvd.percentage.title") + " - " + Messages("summary.renewal"),
     heading = header
 ) {
 
-    <p>@Messages("renewal.percentage.info.line1")</p>
-    <p>@Messages("renewal.percentage.info.line2")</p>
-
-    <ul class="list list-bullet">
-        <li>@Messages("renewal.percentage.info.line3")</li>
-        <li>@Messages("renewal.percentage.info.line4")</li>
-    </ul>
-
-    @legend(Messages("renewal.percentage.turnover.text"))
-
     @form(f, controllers.renewal.routes.PercentageOfCashPaymentOver15000Controller.post(edit)) {
 
         @fieldset(
             f = f("percentage-fieldset"),
-            legend = "renewal.hvd.percentage.title",
-            legendHidden = true,
+            heading = "renewal.hvd.percentage.title",
             panel = false
         ) {
+            <p>@Messages("renewal.percentage.info.line1")</p>
+            <p>@Messages("renewal.percentage.info.line2")</p>
+
+            <ul class="list list-bullet">
+                <li>@Messages("renewal.percentage.info.line3")</li>
+                <li>@Messages("renewal.percentage.info.line4")</li>
+            </ul>
+
+            @legend(Messages("renewal.percentage.turnover.text"))
```

Comment:
why not to use fieldset's legend instead of that?

Reply:
Can't use the fieldset legend as it would mean the fieldset would contain both, which causes voiceover problems.

Reply:
ok, looks good then 👍